pandaharvester/harvesterpreparator/analysis_aux_preparator.py

```diff
@@ -50,8 +50,14 @@ def trigger_preparation(self, jobspec):
                     tmpLog.debug('getting via http from {0} to {1}'.format(url, accPath))
                     res = requests.get(url, timeout=180, verify=False)
                     if res.status_code == 200:
-                        with open(accPath, 'w') as f:
-                            f.write(res.content)
+                        tmpLog.debug('res.headers = {0}'.format(res.headers))
+                        # Should we check the returned size if too small through error?
+                        if res.headers['content-type'] == 'application/gzip' :
```

Comment:
Isn't it enough to open the file in the binary mode? Do you need this if-else?

Reply:
binary mode works with runcontainer transform

Reply:
removed the if else logic in latest commit.